src/test/java/menagerie/model/search/rules/TitleRuleTest.java

```diff
@@ -0,0 +1,47 @@
+package menagerie.model.search.rules;
+
+import menagerie.model.menagerie.GroupItem;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+class TitleRuleTest {
+
+  private GroupItem createGroupItem(String title) {
+    return new GroupItem(null, 1, 1, title);
+  }
+
+  @Test
+  void testAccept() {
```

Comment:
[optional] move the items to class fields and split up the test into 4 separate cases

Reply:
Done